### DISCUSSION TITLE
Copilot won't start

### DISCUSSION THREAD

**Discussion by JoltCode** [community/community#41097] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello, over the past few weeks copilot has been intermittently working, however, when it has not been working it has been showing this error:

`[ERROR] [default] [2022-12-08T13:44:03.642Z] GitHub Copilot could not connect to server. Extension activation failed: "getaddrinfo EAI_AGAIN api.github.com"`

![image](https://user-images.githubusercontent.com/46378904/206462213-e8359621-d052-49c9-b4b7-0a23919b98e3.png)

Any ideas on how I can fix this?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @jolteonyellow! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
